azure/worker/functions.py

```diff
@@ -16,9 +22,9 @@ class FunctionInfo(typing.NamedTuple):
     requires_context: bool
     is_async: bool
 
-    input_binding_types: typing.Mapping[str, str]
-    output_binding_types: typing.Mapping[str, str]
```

Comment:
this looks unintentional, we renamed these *_binding_types

Reply:
nvm

Reply:
Yeah, the new `ParamTypeInfo` struct now contains **both** "binding type name" and "python type", so `input_types` name is a bit more descriptive than `input_binding_types` now.